packages/salesforcedx-vscode-apex/src/apexLanguageClient.ts

```diff
@@ -21,4 +31,62 @@ export class ApexLanguageClient extends LanguageClient {
   public get errorHandler(): ApexErrorHandler | undefined {
     return this._errorHandler;
   }
+
+  public async stop(): Promise<void> {
+    await super.stop();
+  }
+
+  public async showOrphanedProcessesDialog(orphanedProcesses: ProcessDetail[]) {
+    const orphanedCount = orphanedProcesses.length;
+
+    if (orphanedCount === 0) {
+      return;
+    }
+
+    let choice: string | undefined = nls.localize('terminate_show_processes');
+    do {
+      choice = await vscode.window.showWarningMessage(
+        nls.localize(
+          'terminate_orphaned_language_server_instances',
```

Comment:
What do you think about creating constants for string keys? Makes for clean testing and less chance of typo bugs.